code-of-conduct.md

```diff
@@ -8,11 +8,11 @@ The project gains strength from a diversity of backgrounds and perspectives in o
 
 ### Using This Code
 
-This isn’t an exhaustive list of things that you can’t do. Rather, it’s a guide for participation in the community that outlines how each of us can work to keep Apollo a positive, successful, and growing project. 
+This isn’t an exhaustive list of things that you can’t do. Rather, it’s a guide for participation in the community that outlines how each of us can work to keep Meteor a positive, successful, and growing project. 
 
-This code of conduct applies to all spaces managed by the Apollo project or company. This includes Slack, GitHub issues, the GraphQL Summit and GraphQL SF events, and any other forums created by the Apollo team which the community uses for communication. Breaches of this code outside these spaces may affect a person's ability to participate within them. We expect it to be honored by everyone who represents or participates in the project, whether officially or informally.
+This code of conduct applies to all spaces managed by the Meteor project or company. This includes Slack, GitHub issues, and any other forums created by the Meteor team which the community uses for communication. Breaches of this code outside these spaces may affect a person's ability to participate within them. We expect it to be honored by everyone who represents or participates in the project, whether officially or informally.
 
-If you believe someone is violating the code of conduct, please report it by emailing [community@apollodata.com](mailto:community@apollostack.com). 
+If you believe someone is violating the code of conduct, please report it by emailing [community@meteor.com](mailto:community@meteor.com). 
```

Comment:
@abernix @benjamn Is this a valid email address? I just guessed, so let me know if it should be changed to something else.

Reply:
I think it is, but I will check. :)

Reply:
Confirmed, good to use community@meteor.com!